### DISCUSSION TITLE
Release 0.3.7 (1.20.10.23)

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#121] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/118)_ 

--- 

## Summary 

Hello! Thank you for the great feedback in GitHub! We really enjoy chatting with you in those discussions and really appreciate your investment in building alongside us!  

This week we have a few modest updates. After this week, the Preview branch is going to be locked for critical bug fixes for a couple of weeks. Once the Preview branch opens again, we will post updates, the next of which we are very excited about! 🙃 

--- 

## Fixed this release 

- Fixed a bug where error messages related to initial script loading were not being displayed in the log panel. 

- /scriptEvent command can now be used in the Editor without the need to enable the Beta APIs experiments.  

- Fixed a bug where the viewport scaling was not working correctly on devices that use D3D11. We know viewport scaling has been really wacky, and we’ve got concurrent efforts to try and make it better. Please let us know what you think – _especially_ if anything changed for you, in a good or bad way!  

--- 

## Next  

v0.4: The API Update will be focused on exposing the Editor APIs for you to begin building your own Editor extensions. For now we invite you to acquaint yourself with the existing [scripting](https://docs.microsoft.com/en-us/minecraft/creator/scriptapi/) efforts and get familiar with Typescript! 

  

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/TZnQKFwGpt)! 

--- 

Thank you for being here! You're the best! ❤

**Comment by infernostorm** [comment]
Plz bring back editions mode, I need the achievements for packs 2 and 5
